src/MongoDB.Driver.Core/Core/Authentication/ScramShaAuthenticator.cs

```diff
@@ -30,12 +30,14 @@ namespace MongoDB.Driver.Core.Authentication
     /// </summary>
     public abstract class ScramShaAuthenticator : SaslAuthenticator
     {
+#pragma warning disable CA1724
```

Comment:
Yeah, agree.

Reply:
I would find a way to avoid the name conflict rather than to suppress the warning.

Is it due to a using statement?

Reply:
This is due to a bug in the FxCopAnalyzers.

You can remove the #pragma if you upgrade FxCopAnalyzers from 2.6.1 to 2.6.2.

Don't try to upgrade any further (or much further) than 2.6.2 because newer versions are stricter and we get more warnings/errors.